### DISCUSSION TITLE
Blocks for the website builder

### DISCUSSION THREAD

**Discussion by MattinNajafi** [payloadcms/payload#9428] [Q&A]
Hi! Reading the documentation shows these blocks: 

<img width="774" alt="Skärmavbild 2024-11-22 kl  10 23 35" src="https://github.com/user-attachments/assets/12c3104f-01a2-4932-9d12-8e40c4ca2708">

But when i started my own local testing environment running `npx create-payload-app ` i just had these:

<img width="1647" alt="Skärmavbild 2024-11-22 kl  10 23 55" src="https://github.com/user-attachments/assets/31d0ce55-4d33-4f4c-b940-3d2c40f7322d">

What can I do to get all supported blocks? Is there any custom blocks out there to import aswell that is getting rendered out by <RenderBlocks/> component?

**Comment by mikevarela** [comment]
I'm curious about this too, I'm using the website template, downloaded yesterday, and see the same few block, missing the others.

**Comment by sylkenio** [comment]
@MattinNajafi @mikevarela The first image appears to be more similar to the blocks used on their official website. They might've used that image to show what can be done.

You can find their blocks for their official site at https://github.com/payloadcms/website

Just keep in mind, their structure isn't the same as the website template. 

So, inside of their `src/blocks/*` directory is basically where you'll find the equivalent of the `config.ts` files in the website template. Then inside of `src/components/blocks/*` is where you will find the actual block components. If you don't mind inspecting their code and reimplementing it to fit your needs, you could use their custom block components.

Just be warned, a lot of their official site is deeply nested inside of itself, so copy and pasting most of their block components won't work. 

If this is the route you want to go, I suggest sticking to the config.ts and Component.tsx structure provided in the website template, as it's much easier to maintain and utilize.

Hope this helps you find what you're looking for!

P.S. : I haven't checked any of the community repos, there may be some repos out there that the community has available that comes with more custom block components.

I am currently working on releasing a community template that starts out with their custom responsive Grid/Gutter setup.
